test/hclust.jl

```diff
@@ -1,3 +1,5 @@
+using Clustering
```

Comment:
If that was needed I guess the tests would fail?

Reply:
The test as run from `src/runtests.jl` would work all right, but this line allows a separate test
```julia
include("src/Clustering.jl")
include("test/hclust.jl")
```

Reply:
I assume this isn't the cause of the nightly failure?
```
ERROR: LoadError: LoadError: importing Clustering into Main conflicts with an existing identifier
```

Reply:
All the other tests in `test/` have `using Clustering` in their code (but after `using Base.Test`---I don't assume order can matter here).  

On my machine, it is `mcl.jl` that fails on an `InexactError()`.  But that is a different issue I suppose.

Reply:
Huh, weird. Okay.

Reply:
I'd check whether the error hasn't been introduced by this PR though, and if so file an issue in Base.